testcafe/tests/pages/drive-model.js

```diff
@@ -0,0 +1,83 @@
+import { Selector, t } from 'testcafe'
+import {
+  getElementWithTestId,
+  getPageUrl,
+  checkExistVisible
+} from '../helpers/utils'
+
+export default class Page {
+  constructor() {
+    //Sidebar
+    this.sidebar = getElementWithTestId('driveSidebar')
+    this.btnNavToFolder = getElementWithTestId('navToFolder')
+    this.btnNavToRecent = getElementWithTestId('navToRecent')
+    this.btnNavToSharing = getElementWithTestId('navToSharing')
+    this.btnNavToTrash = getElementWithTestId('navToTrash')
+    this.breadcrumbTitle = getElementWithTestId('path-title')
+    //Main menu
+    this.cozBar = Selector('#coz-bar')
+    this.btnMainApp = this.cozBar.find('div.coz-nav-apps-btns > button')
+    this.cozNavPopContent = Selector('[class*=coz-nav-pop]')
+    this.btnCozBarDrive = this.cozNavPopContent
+      .find('li.coz-nav-apps-item > a')
+      .withText('Cozy Drive') // !FIXME: do not use text
+
+    //Error and empty folders
+    // !FIXME better selector for error
+
+    //c-empty is use for empty drive, or error..
+    this.driveEmpty = Selector('[class*="c-empty"]')
+      .parent(0)
+      .withAttribute('data-test-id', 'fil-content-body')
+
+    //oops is a class for error only
+    this.errorOops = Selector('[class*="oops"]')
+
+    this.errorEmpty = Selector('[class*="c-empty"]')
+      .child('h2')
+      .withText('Switch online!') // !FIXME: do not use text
+  }
+
+  async checkSidebarButton(btn, path, title) {
+    checkExistVisible(btn, `Button ${title}`)
+
+    await t.expect(btn.getAttribute('href')).contains(path)
+    await this.navigateToPage(btn, path, title)
+
+    console.log(`Navigation using Button ${title} OK!`)
+  }
+
+  async navigateToPage(btn, path, title) {
+    await t
+      .click(btn)
+      .expect(getPageUrl())
+      .contains(path)
+
+    checkExistVisible(this.breadcrumbTitle, `Page title : ${title}`)
+
+    await t.expect(this.breadcrumbTitle.child('span').innerText).contains(title)
```

Comment:
It'll break in a near future since we'll rework this component. And I don't think we will have a span anymore.

Reply:
Should I add a `data-test-id` in the `span` for the moment so you'll move it to the new componnant displaying the title ?
```
  <span
     data-test-id="path-link-name"
     className={styles['fil-path-link-name']}
     >
          {folder.name}
   </span>
```